exprconv/expr.hpp

```diff
@@ -58,4 +68,44 @@ class ExprtkConverter : public DataConverter {
             int dummy = (int)(val * divider);
             return dummy / divider;
         }
+
+        static double int32be(const double register1, const double register2) {
+            return ConverterTools::toNumber<int32_t>(register1, register2);
+        }
+
+        static double int32le(const double register1, const double register2) {
+            return ConverterTools::toNumber<int32_t>(register1, register2, true);
+        }
+
+        static double uint32be(const double register1, const double register2) {
+            return ConverterTools::toNumber<uint32_t>(register1, register2);
+        }
+
+        static double uint32le(const double register1, const double register2) {
```

Comment:
I do not think that be/le versions are needed in int32 case. Byte order for int16/uint16 register value is always correct, libmodbus takes care about it. I do not this that there is any real example of int16 value encoded in reverse (big endian order).

I would just add int32() and uint32(). In the future we can add int32be(r1,r2) if there is any real example of such conversion.

We need a documentation which part of int32 are r1 and r2 arguments: (r1 - high part , r2- low part)

Reply:
OK, done.